src/configuration/configuration.json

```diff
@@ -0,0 +1,12 @@
+{
+    "plugin_preferences":{
+        "kubernator": "ollama"
```

Comment:
bad plugin name, plugin name is `kubernator-plugin`

Reply:
removed